docs/source/release_notes/release-notes.rst

```diff
@@ -44,8 +44,14 @@ develop
     *    - Type
          - Change
 
+    *    - |fixed|
+         - KVS migrations will no longer verify equality between the from and to KVSes for the sweep priority and progress tables.
+           Note that these tables are still *migrated* across, as they provide heuristics for timely sweeping of tables.
+           However, these tables are expected to change as the migration process performs writes to the to-KVS.
```

Comment:
Yes, the `from-kvs` could be swept.

Regarding the `to-kvs`, there won't be anything to sweep, but my concern was that the write counts in the `to-kvs` would be affected by writing?